tasks-api/src/main/java/org/infinispan/tasks/ServerTask.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.tasks;
+
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.concurrent.Callable;
+
+/**
+ * Author: Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
```

Comment:
Since this is a part of public API, I would suggest adding some meaningful Javadoc here (with a description how it works etc).